### PR TITLE
Remove google plus

### PR DESCRIPTION
**Pull request by therealjuan**
1. Pull the branch or run the demo service
2. Check on every page there is no Google Plus in the footer
3. #winning

### CODE REVIEW

**Comment by nottrobin** [comment]
What a historical moment!

File: src/common/style/vanilla/footer.scss [hunk 1]
```diff
@@ -71,11 +71,6 @@ body,
     background-position: 0 0;
   }
 
-  .p-social-icon--google {
-    @extend %p-social-icon;
-    background-position: -40px 0;
-  }
-
```
Comment (cjwatson) [inline src/common/style/vanilla/footer.scss]: Don't you need to move the `background-position` for Facebook and Youtube to account for this?  They seem to be positioned at 40-pixel intervals.

**Comment by bartaz** [comment]
We should update footer layout to have narrower space for icons - not what there are less of them:

screenshot 2019-02-01 at 12 07 34

Changing from col-8/col-4 to col-9/col-3 should be enough.

File: src/common/style/vanilla/footer.scss (hunk 1 above)
Comment (bartaz) [reply src/common/style/vanilla/footer.scss]: The sprite image containing the icons doesn't change, so the position of other icons is not changed.

**Comment by therealjuan** [comment]
Changing to col-8 and col-4

**Comment by nottrobin** [comment]
@therealjuan could you fix the lint errors:

```
/home/travis/build/canonical-websites/build.snapcraft.io/src/common/components/footer/index.js
  62:94   error  Unknown property 'accesskey' found, use 'accessKey' instead  react/no-unknown-property
  65:107  error  Unknown property 'accesskey' found, use 'accessKey' instead  react/no-unknown-property
  68:93   error  Unknown property 'accesskey' found, use 'accessKey' instead  react/no-unknown-property
  71:96   error  Unknown property 'accesskey' found, use 'accessKey' instead  react/no-unknown-property
  74:136  error  Unknown property 'accesskey' found, use 'accessKey' instead  react/no-unknown-property
  74:152  error  HTML entities must be escaped                                react/no-unescaped-entities
✖ 6 problems (6 errors, 0 warnings)
  5 errors and 0 warnings potentially fixable with the `--fix` option.
```

**Comment by therealjuan** [comment]
@bartaz  👍 🎉 :shipit:  👍 🎉 :shipit:  LGTM 👍 🎉 :shipit:  👍 🎉 :shipit:  👍 🎉 :shipit:

**Pull request merged**